hs_access_control/models.py

```diff
@@ -1277,6 +1277,23 @@ def can_change_resource_flags(self, this_resource):
 
         return self.user.is_superuser or (not this_resource.raccess.published and self.owns_resource(this_resource))
 
+    def can_create_resource_version(self, this_resource):
+        """
+        Whether self can create a version of a resource.
+
+        :param this_resource: Resource to check.
+        :return: True if user can set create a version otherwise false.
+
+        This is not enforced. It is up to the programmer to obey this restriction.
+        """
+
+        if __debug__:  # during testing only, check argument types and preconditions
+            assert isinstance(this_resource, BaseResource)
+
+        if not self.user.is_active: raise PermissionDenied("Requesting user is not active")
+
+        return self.owns_resource(this_resource)
+
```

Comment:
@pkdash Do we really need to add this function given that we only allow users to version a resource? Can we just use self.owns_resource() to check for whether a user can create a new version of a resource or not? user.is_active() check should apply to any privilege check since we don't allow inactive users to do anything, right?